v8/orgmode/README.md

```diff
@@ -44,3 +44,7 @@ because the path is considered as an absolute file path.
 
 In order to correctly generate image urls, you may write `[[img-url:/images/test.jpg]]`,
 and then it should be generated as `<img src="/images/test.jpg" alt="test.jpg">`.
+
+The internal link in ox-html is a little fuzzy. For example, `[[./test.org][test]]` will be generated as `<a href="file:///./test.jpg">"test"` because the path is considered as an absolute file path.
```

Comment:
I don’t think you meant `.jpg` here.

Reply:
Indeed forgot to change that one... I was tired after studying how it all worked